### ISSUE TITLE
SMART and drive statistics are unavailable for drives used by ZFS

### ISSUE THREAD

**Issue by d-uzlov** [openzfsonwindows/openzfs#240]
### System information

```powershell
PS C:\Windows\system32> zfs version
zfswin-2.2.0rc1
zfs-kmod-zfswin-2.2.0rc1

PS C:\Windows\system32> [System.Environment]::OSVersion

Platform ServicePack Version      VersionString
-------- ----------- -------      -------------
 Win32NT             10.0.19045.0 Microsoft Windows NT 10.0.19045.0
```

### Describe the problem you're observing

As soon as I run `zpool create` with physical drives, I can no longer observe their SMART values.

Also Task Manager doesn't show drive statistics (active time, read/write speed).
I guess these 2 issues must have the same cause.

If I do `zpool export` and then `zpool import`, SMART issues are fixed.
But Task Manager still doesn't show drive statistics.

### Describe how to reproduce the problem

Open CrystalDiskInfo, look at the list of drives.

Run `zpool create tank PHYSICALDRIVE0`.

Do `Menu Function -> Rescan` in CrystalDiskInfo. Drive that is used by ZFS has disappeared.

**Comment by lundman** [comment]
Yeah, we found that initially we just used the disks in the pool, but there was nothing stopping some other process/program from using the "apparently" available disks, even "zpool create" again.
So as a temporary measure, we changed it to "lock" the disks, by making them hidden. This most likely is why SMART/stats cant see them.

Eventually the idea was we would figure out how we SHOULD make it clear the disk is "busy", but also not lock it "too much".

If anyone knows, let me know :)
